homeassistant/components/sensor/forecast.py

```diff
@@ -56,6 +59,16 @@
                              'mm', 'in', 'mm', 'mm', 'mm'],
 }
 DEFAULT_NAME = "Forecast.io"
+CONF_UNITS = 'units'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_MONITORED_CONDITIONS):
+        vol.All(cv.ensure_list, [vol.In(list(SENSOR_TYPES.keys()))]),
```

Comment:
The default iterator of a dictionary is iterating through the keys. Thus `.keys()` can be omitted.